pygen/pygen_src/isa/riscv_instr.py

```diff
@@ -62,20 +64,15 @@ def __init__(self):
         self.is_hint_instr = None
         self.is_floating_point = None
         self.imm_str = None
-        self.comment = None
-        self.label = None
+        self.comment = ""
+        self.label = ""
         self.is_local_numeric_label = None
         self.idx = -1
         self.has_rs1 = 1
         self.has_rs2 = 1
         self.has_rd = 1
         self.has_imm = 1
-
-        # Fields Added for debugging These fields are actually from a different files.
-        self.unsupported_instr = []
-        self.XLEN = 32
-        self.supported_isa = [riscv_instr_group_t.RV32I]
-        self.implemented_csr = []
+        self.shift_t = 2 ** 32 - 1
```

Comment:
similarly, should this value be a BitArray?
We should still be able to use the binary operators on BitArray structures as well.